app/getRequests.tsx

```diff
@@ -0,0 +1,11 @@
+let requestsArr:Array<Object> = [{url : "", requestBody: undefined , method : "", type: ""}]
+
+chrome.webRequest.onBeforeRequest.addListener(
+function(details) {
```

Comment:
Try spend some time setting up your editor. And try using https://github.com/prettier/prettier

Reply:
Sure Kashyap. Will do that and follow .editorconfig